source/client.js

```diff
@@ -410,6 +410,22 @@ export class ImmersClient extends window.EventTarget {
     return this.activities.note(DOMPurify.sanitize(content), to, privacy)
   }
 
+  /**
+   * Delete  user's avatar from their Avatar Collection.
+   * @param  {(object|string)} sourceActivity - IRI of activity or Activity in the Outbox
+   * @returns {Promise<string>} IRI of the remove activity
+   */
+  async deleteMessage (sourceActivity) {
+    switch (sourceActivity.type.toLowerCase()) {
+      case 'arrive':
+        return this.activities.undo(sourceActivity)
+      case 'leave':
+        return this.activities.undo(sourceActivity)
```

Comment:
when you've got multiple identical cases, I prefer to use fallthrough to avoid code duplication. The linter will let you do this as long as the cases statements are stacked like this on a single code block

```suggestion
      case 'arrive':
      case 'leave':
        return this.activities.undo(sourceActivity)
```